Environmental Sensing/Radon Sensor Signoff.md

```diff
@@ -0,0 +1,75 @@
+# Radon Sensor Signoff
+
+# Function of the subsystem
```

Comment:
Be brief.

Reply:
The purpose of this section is to state the function. Don't talk about components. Don't talk about Radon in general. 

State clearly what this subsystem is to do and nothing more.